medical_emr/__manifest__.py

```diff
@@ -39,7 +40,9 @@
     'version': '8.0.1.1',
     'author': "Odoo Medical Team, LasLabs, Odoo Community Association (OCA)",
     'category': 'Medical',
-    'depends': ['medical'],
+    'depends': [
```

Comment:
`medical_emr` is a deprecated module & is being stripped for parts. As such, the only changes that we should see in this module is the removal of logic that is being used elsewhere. We should not be adding anything.